redGreen.js

```diff
@@ -26,6 +31,7 @@ const fetchData = async () => {
   return data;
 };
 
+
 const fetchQuotes = async () => {
```

Comment:
These can be combine into a single function just by putting 27 and 28 into fetchQuotes